apis/registries/v1alpha2/project_types.go

```diff
@@ -52,22 +52,26 @@ type ProjectSpec struct {
 type ProjectMetadata struct {
 	// Whether content trust is enabled or not
 	// If it is, users can not pull unsigned images from this project
-	EnableContentTrust bool `json:"enableContentTrust"`
+	// +kubebuilder:validation:Optional
+	EnableContentTrust bool `json:"enableContentTrust,omitempty"`
```

Comment:
Just to see if I understood correctly: A ProjectMetadata custom resource should be valid when `EnableContentTrust` isn't set but is implicitly false, right? Same for `AutoScan`, `ReuseSysCVEAllowlist` and `PreventVul`.

Reply:
A `ProjectMetadata` is contained as an optional field in a `Project` custom resource's `Spec.Metadata`.
Yet you are right, the `Project` custom resource should be valid, if all the `ProjectMetadata` fields you listed are unset.
The only _required_ field for project creation is `ProjectMetadata.Public`.